spec/bolt/cli_spec.rb

```diff
@@ -1989,31 +2003,31 @@ def stub_config(file_content = {})
 
     it 'reads private-key for ssh' do
       with_tempfile_containing('conf', YAML.dump(complete_config)) do |conf|
-        cli = Bolt::CLI.new(%W[command run --configfile #{conf.path} --nodes foo --no-host-key-check])
+        cli = Bolt::CLI.new(%W[command run uptime --configfile #{conf.path} --nodes foo --no-host-key-check])
         cli.parse
         expect(cli.config.transports[:ssh]['private-key']).to eq('/bar/foo')
       end
     end
 
     it 'reads host-key-check for ssh' do
       with_tempfile_containing('conf', YAML.dump(complete_config)) do |conf|
-        cli = Bolt::CLI.new(%W[command run --configfile #{conf.path} --nodes foo])
+        cli = Bolt::CLI.new(%W[command run uptime --configfile #{conf.path} --nodes foo])
         cli.parse
         expect(cli.config.transports[:ssh]['host-key-check']).to eq(false)
       end
     end
 
     it 'reads run-as for ssh' do
       with_tempfile_containing('conf', YAML.dump(complete_config)) do |conf|
-        cli = Bolt::CLI.new(%W[command run --configfile #{conf.path} --nodes foo --password bar --no-host-key-check])
+        cli = Bolt::CLI.new(%W[command run r --configfile #{conf.path} --nodes foo --password bar --no-host-key-check])
```

Comment:
Is this an actual command?

Reply:
It doesn't need to be an actual command. The reason I used a single character is to avoid line length rubocop

Reply:
Ah, makes sense :+1: